game/project.godot

```diff
@@ -20,6 +20,7 @@ config/project_settings_override.template="user://settings.cfg"
 
 Events="*res://src/Autoload/Events.gd"
 Resolution="*res://src/Autoload/Resolution.gd"
+MusicConductor="*res://src/MusicConductor/MusicConductor.tscn"
```

Comment:
Consider moving what you need to autoload to the Autoload directory, just to keep it very clean with what is where.

Reply:
As mentioned as part of Orwellian's Project Structure, we should keep scenes and their scripts together
https://github.com/OpenVic2Project/OpenVic2/pull/14